backend/sirius-web-forms/src/main/java/org/eclipse/sirius/web/forms/components/CheckboxComponent.java

```diff
@@ -47,11 +49,14 @@ public Element render() {
         BiFunction<VariableManager, Boolean, Status> genericHandler = checkboxDescription.getNewValueHandler();
         Function<Boolean, Status> specializedHandler = newValue -> genericHandler.apply(variableManager, newValue);
 
+        List<Element> children = new DiagnosticRendererUtil().renderDiagnostics(checkboxDescription, variableManager);
```

Comment:
Somehow I missed that during my first review but this is equivalent to the following code:

```
const CheckboxComponent = () => {
  return (
    <checkbox id={id} label={label} value={value.booleanValue()} newValueHandler={specializedHandler}>
      {new DiagnosticRendererUtil().renderDiagnostics(checkboxDescription, variableManager)}
    </checkbox>
  );
};
```

Use components instead with something like that:

```
const CheckboxComponent = () => {
  return (
    <checkbox id={id} label={label} value={value.booleanValue()} newValueHandler={specializedHandler}>
      <DiagnosticComponent description={checkboxDescription} variableManager={variableManager} />
    </checkbox>
  );
};
```

It makes no sense to do that in a React based approach. A function which returns elements is a component. As I said earlier, do not submit a review with something called `Util`, you are doing something wrong.